pywatts/modules/wrappers/sklearn_wrapper.py

```diff
@@ -105,17 +105,19 @@ def transform(self, **kwargs: xr.DataArray) -> xr.DataArray:
         :return: the transformed output
         """
         x_np = self._dataset_to_sklearn_input(kwargs)
+        targets = self.targets
 
         if isinstance(self.module, TransformerMixin):
             prediction = self.module.transform(x_np)
+            targets = []  # output of transformer must not match the shape of the target
         elif "predict" in dir(self.module):
             prediction = self.module.predict(x_np)
         else:
             raise KindOfTransformDoesNotExistException(
                 f"The sklearn-module in {self.name} does not have a predict or transform method",
                 KindOfTransform.PREDICT_TRANSFORM)
 
-        return self._sklearn_output_to_dataset(kwargs, prediction, self.targets)
+        return self._sklearn_output_to_dataset(kwargs, prediction, targets)
 
     def inverse_transform(self, **kwargs: xr.DataArray) -> xr.DataArray:
         """
```

Comment:
needs also be implemented for inverse_transform

Reply:
The selector Mixin has inverse transform methods. So please test if this is necessary.